### PR TITLE
Remove TableTools.emptyTable overload that takes a definition 

### LINKED ISSUE #752: emptyTable should not accept a table definition
This causes some annoyance in creating columns populated only with null values, with no safe way to change those values, except operations that would actually create the columns to begin with, making this operation entirely unnecessary.

This existed for legacy reasons, and is not needed in this repository. This change will affect TableTools and the gRPC api.

### PR DESCRIPTION
**Pull request by niloc132**
This doesn't really make any sense unless the caller is assuming that they will be able to write to the column sources once it is created, and presents a confusing API to users.

Fixes #752

### CODE REVIEW

**Comment by niloc132** [comment]
TableBuilder is now deprecated (and should be replaced by one of our input table variants at some point?), but I confirmed that current main and this branch produce the same table.parquet output for the tests in SumByBenchmark as a way to validate these changes.

**Comment by niloc132** [comment]
Do not merge - need to update py, and can't do that without having java15 installed...

**Comment by niloc132** [comment]
> Do not merge - need to update py, and can't do that without having java15 installed...

Done.

File: grpc-api/src/main/java/io/deephaven/grpc_api/table/ops/EmptyTableGrpcImpl.java [hunk 1]
```diff
@@ -31,84 +31,17 @@ public EmptyTableGrpcImpl() {
...
-        if (types.size() != names.size()) {
-            throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "Inconsistent number of column names and types.");
+        if (request.getSize() < 0) {
+            throw GrpcUtil.statusRuntimeException(Code.INVALID_ARGUMENT, "Size must be greater than zero");
         }
     }
 
     @Override
     public Table create(final EmptyTableRequest request, final List<SessionState.ExportObject<Table>> sourceTables) {
         Assert.eq(sourceTables.size(), "sourceTables.size()", 0);
```
Comment (rcaudy) [inline grpc-api/src/main/java/io/deephaven/grpc_api/table/ops/EmptyTableGrpcImpl.java:43]: `eqZero` is even more idiomatic, or `Assert.assertion(sourceTables.isEmpty(), "sourceTables.isEmpty()")`

**Review by rcaudy — APPROVED** [review]

**Pull request merged**